src/hnsw/options.c

```diff
@@ -12,6 +12,12 @@
 #include <utils/rel.h>  // RelationData
 #include <utils/syscache.h>
 
+#ifdef _WIN32
```

Comment:
What is this about? did you test it on windows?!

Reply:
I didn't as I don't have windows machine, but I found it on stackoverflow that the `unistd.h` does not exist in windows. https://stackoverflow.com/questions/37437082/which-is-a-good-practice-to-use-access-function